VAMobile/src/components/Nametag/Nametag.test.tsx

```diff
@@ -20,12 +20,12 @@ context('Nametag', () => {
         personalInformation: {
           ...InitialState.personalInformation,
           profile: {
-            preferredName: 'Benny',
+            preferredName: '',
```

Comment:
I know this might be nitpicking but is this and the genderIdentity field even needed anymore? Should we remove these as we go or will this be all removed once everything gets converted over. Lastly I think we forgot to ask how we mock the api call results for demo mode with the new way of doing things. Would it be the same?